CI/flatpak/com.obsproject.Studio.json

```diff
@@ -144,9 +144,25 @@
       ],
       "sources": [
         {
-          "type": "archive",
-          "url": "https://www.ffmpeg.org/releases/ffmpeg-4.3.2.tar.xz",
-          "sha256": "46e4e64f1dd0233cbc0934b9f1c0da676008cad34725113fb7f802cfa84ccddb"
+          "type": "git",
+          "url": "https://git.ffmpeg.org/ffmpeg.git",
+          "tag": "n4.4.1",
+          "commit": "7e0d640edf6c3eee1816b105c2f7498c4f948e74"
```

Comment:
Perhaps it would be better to switch to https://github.com/obsproject/FFmpeg/commits/v4.4-plus-extra-fixes ? I'm not sure, needs double checking.

Reply:
The additional commit from this branch are put as patches in obs-deps.